.github/workflows/check-ui-links.yaml

```diff
@@ -0,0 +1,75 @@
+name: Check links from the UI
+
+on:
+  pull_request:
+    branches:
+      - master
+  schedule:
+    - cron: '0 1 * * *' # once a day at 1 am
+    
+jobs:
+  preview:
+    name: Check links from UI
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check-out
+        uses: actions/checkout@v3
+
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          working-directory: docs
+          ruby-version: '2.7'
+          bundler-cache: true 
+
+      - name: Build latest
+        working-directory: docs
+        run: bundle exec jekyll build --config _config.yml -d _site/ --verbose
+
+      - name: Get links to check
+        id: get-links
+        run: |
+          escaped_pwd=$(pwd | sed 's/[\/&]/\\&/g')
+          echo $pwd
+          echo $escaped_pwd
+          ls -l 
+
+          find webui/src -type f ! -name "*.js" -exec grep -Eo 'href="([^"]*)"' {} \; | \
+            cut -d'"' -f2 | \
+            grep -v -e "^/" -e "^#" | \
+            sed "s/https:\/\/docs.lakefs.io/file:\/\/$escaped_pwd\/docs\/_site/" > /tmp/links_to_check.txt
```

Comment:
Can we give lychee the _site to verify all links?

Reply:
this is already handled by the [docs PR action](https://github.com/treeverse/lakeFS/blob/master/.github/workflows/docs-pr.yaml#L58)

Reply:
So why do we need a special way to extract the docs links and not just render and verify as we do for the PR?

Reply:
This PR is for checking links from within the product, which need handling differently because of the absolute URLs.
The existing action is from links within docs. 
I guess we could logically combine the two into one with separate steps but I'd rather do that as a future iteration.